stanford_image.strongarm.inc

```diff
@@ -175,6 +175,18 @@ function stanford_image_strongarm() {
       'stanford_huge_landscape' => array(
         'custom_settings' => FALSE,
       ),
+      'stanford_banner_850x400' => array(
+        'custom_settings' => TRUE,
+      ),
+      'gallery_large' => array(
```

Comment:
Since this branch was deleted, I put it here:
https://github.com/SU-SWS/stanford_image/pull/38